llvm/lib/Transforms/Instrumentation/HWAddressSanitizer.cpp

```diff
@@ -187,10 +187,8 @@ static cl::opt<bool>
                               cl::desc("Use selective instrumentation"),
                               cl::Hidden, cl::init(false));
 
-static cl::opt<int> ClHotPercentileCutoff(
-    "hwasan-percentile-cutoff-hot", cl::init(0),
-    cl::desc("Alternative hot percentile cuttoff."
-             "By default `-profile-summary-cutoff-hot` is used."));
+static cl::opt<int> ClHotPercentileCutoff("hwasan-percentile-cutoff-hot",
```

Comment:
should it be an error to give `hwasan-selective-instrumentation` but neither `hwasan-percentile-cutoff-hot` nor `hwasan-random-skip-rate"`?

Reply:
I want to remove hwasan-selective-instrumentation #87709